### DISCUSSION TITLE
Creación y gestión en línea de repositorios y documentos en GitHub

### DISCUSSION THREAD

**Discussion by rcfdtools** [rcfdtools/R.TeachingResearchGuide#7] [General]
https://github.com/rcfdtools/R.TeachingResearchGuide/tree/main/Section01/GitHubRepository

En esta actividad aprenderá a crear y gestionar repositorios en GitHub directamente desde el navegador de Internet.

_Realice o publique aquí consultas, ideas, comentarios, preguntas y respuestas relacionadas con esta actividad._

**Comment by juanrodace** [comment]
A la pregunta: ¿Debo realizar algún pago por los servicios de GitHub?
- Respuesta: No, existe la opción de pago si quiero tener archivos de más de 100Mb en mi repositorio.

A la pregunta: ¿Que pasa con mis repositorios si accidentalmente elimino mi cuenta de usuario?
 - Respuesta: Toda la información quedaría almacenada en el disco duro del equipo de computo en el que estoy trabajando.

**Reply by rcfdtools** [reply]
@juanrodace, gracias por tus aportes. Parte de tus comentarios han sido incluídos en la sección Q&A de https://github.com/rcfdtools/R.TeachingResearchGuide/tree/main/Section01/GitHubRepository
